docs/kernel_troubleshooting.rst

```diff
@@ -0,0 +1,183 @@
+Troubleshooting Kernel Updates
+==============================
+For security reasons, new versions of the Linux kernel running on your your 
```

Comment:
Suggest a bit more clarity here: Kernel updates address known bugs and security vulenrabilities in the Linux kernel. It is important for your SecureDrop instance to run the latest version.

Reply:
reworded